src/components/ErrorMessages/ErrorMessages.module.css

```diff
@@ -1,4 +1,4 @@
 .warning {
     color: red;
     font-size: 12px;
-}
+}
```

Comment:
This file needs formatting try running `npm run format`

Reply:
Done.